Mathlib/Data/ByteArray.lean

```diff
@@ -9,11 +9,16 @@ import Mathlib.Init
 # Main result
 Introduce main properties of `Up` (well-ordered relation for "upwards" induction on `ℕ`) and of
  `ByteArray`
+
+This entire file has been deprecated on 2024-08-19 in favour of `ByteSubarray` in Batteries.
 -/
 
+set_option linter.deprecated false
+
 namespace Nat
 
 /-- A well-ordered relation for "upwards" induction on the natural numbers up to some bound `ub`. -/
+@[deprecated (since := "2024-08-19")]
```

Comment:
```suggestion
@[deprecated "deprecated in favour of `ByteSubarray` in Batteries"  (since := "2024-08-19")]
```
Can we leave a message like this?

Reply:
I don't think that makes any sense here, since `Up` is unrelated to `ByteSubarray`.